core/src/test/java/dk/alexandra/fresco/suite/dummy/arithmetic/AbstractDummyArithmeticTest.java

```diff
@@ -77,6 +77,9 @@ private void runTest(
       TestThreadRunner.TestThreadFactory<DummyArithmeticResourcePool, ProtocolBuilderNumeric> f,
       EvaluationStrategy evalStrategy, int noOfParties, BigInteger mod, int maxBitLength,
       int fixedPointPrecision, boolean logPerformance) {
+    if (evalStrategy == EvaluationStrategy.SEQUENTIAL) {
+      throw new IllegalArgumentException(":(");
```

Comment:
Maybe pass a more useful string argument to this exception?

Reply:
Actually, maybe not throw an exception?

Reply:
Ah, good catch, forgot to delete that.